pkg/rpc/commands.go

```diff
@@ -98,7 +101,7 @@ var sendBidTx = func(s *Server, params []string) (string, error) {
 		return "", fmt.Errorf("missing parameters: amount/locktime")
 	}
 
-	amount, err := strconv.Atoi(params[0])
+	amount, err := strconv.Atoi(params[1])
```

Comment:
This doesn't seem right

Reply:
yeah, looks like a vim creep. Will change